applications_tests/gls_sharp_navier_stokes_3d/pw_contact_test.output

```diff
@@ -0,0 +1,90 @@
+Running on 1 MPI rank(s)...
+   Number of active cells:       1600
+   Number of degrees of freedom: 8228
+   Volume of triangulation:      1600
+   Number of active cells:       1768
+   Number of degrees of freedom: 9276
+   Volume of triangulation:      1600
+   Number of active cells:       2552
+   Number of degrees of freedom: 13440
+   Volume of triangulation:      1600
+   Number of active cells:       2552
+   Number of degrees of freedom: 13440
+   Volume of triangulation:      1600
+
```

Comment:
It is confusing for me why there are this many repetitions of number of active cells, degrees of freedom .... It would be a good idea to give a small description of what each one of them means